src/api/data/tree/exportApi.ts

```diff
@@ -71,7 +74,7 @@ export class ExportApi {
     const { outputDir, plan, query } = this.config;
 
     if (outputDir) {
-      this.setupOutputDirectory(outputDir);
+      ensureDirSync(outputDir);
```

Comment:
This is being called in an `async` function. Is there a reason for choosing a `Sync` variant of `ensureDir`?